packages/store-api/src/typeDefs/query.graphql

```diff
@@ -0,0 +1,72 @@
+type StoreProductEdge {
+  node: StoreProduct!
+  cursor: String!
```

Comment:
What is this `cursor`? I understand `node`s and `edge`s are representing a graph, but I didn't get this `cursor`

Reply:
The cursor is a way to paginate the API. Was proposed by Relay. You can read about the pro and cons [here](https://www.prisma.io/docs/concepts/components/prisma-client/pagination#cursor-based-pagination).

Reply:
Yes! we are following the [relay specification](https://relay.dev/graphql/connections.htm). This is nice beacuse there are some tools that can do automatic pagination once you follow this spec

Reply:
Awesome!! I was not aware of this pattern 😄